docs/content/how-to/kubevirt/create-kubevirt-cluster.md

```diff
@@ -4,228 +4,226 @@ Install a nested OCP cluster running on VMs within a management OCP cluster
+* The management OCP cluster must have wildcard dns routes enabled.
 
-    Install it using Go 1.18+:
-       ```shell
-       go get -u github.com/openshift/hypershift@latest
        ```
+       oc patch ingresscontroller -n openshift-ingress-operator default --type=json -p '[{ "op": "add", "path": "/spec/routeAdmission", "value": {wildcardPolicy: "WildcardsAllowed"}}]'
+       ```
+* The management OCP cluster must have Openshift Virtualization installed on it. [Instructions for installing Openshift Virtualization](https://docs.openshift.com/container-platform/4.12/virt/about-virt.html?extIdCarryOver=true&sc_cid=7013a0000026OSTAA2)
+* The Management OCP cluster must be configured with OVNKubernetes as the default pod network CNI.
+* The Management OCP cluster must have LoadBalancer service support. [Instructions for installing MetalLB](https://docs.openshift.com/container-platform/4.12/networking/metallb/metallb-operator-install.html)
+* The management OCP cluster must have default storage class. [Storage Configuration Documentation](https://docs.openshift.com/container-platform/4.12/post_installation_configuration/storage-configuration.html)
+    Below is an example of how to annotate a storageclass as the default storageclass.
 
-* Admin access to an OpenShift cluster (version 4.8+) specified by the `KUBECONFIG` environment variable.
-* The management OCP cluster must have Openshift Virtualization installed on it (More info:
-  https://docs.openshift.com/container-platform/4.10/virt/about-virt.html?extIdCarryOver=true&sc_cid=7013a0000026OSTAA2)
-* The management OCP cluster must have valid storage class (More info:
-  https://docs.openshift.com/container-platform/4.10/post_installation_configuration/storage-configuration.html)
+       ```
+       oc patch storageclass ocs-storagecluster-ceph-rbd -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"true"}}}'
+       ```
 * The OpenShift CLI (`oc`) or Kubernetes CLI (`kubectl`).
-* A valid [pull secret](https://cloud.redhat.com/openshift/install/aws/installer-provisioned) file for the `quay.io/openshift-release-dev` repository.
-* A base domain that can be [configured for ingress](#handling-ingress-and-dns)
+* A valid [pull secret](https://console.redhat.com/openshift/install/platform-agnostic/user-provisioned) file for the `quay.io/openshift-release-dev` repository.
 
-## Before you begin
+## Installing HyperShift Operator and hypershift cli tool
 
-Install HyperShift into the management cluster:
+Before creating a guest cluster, the Hypershift operator and hypershift cli tool
+must be installed.
 
-```shell linenums="1"
+The command below builds latest hypershift cli tool from source.
+
+```shell
+podman run --rm --privileged -it -v \
+$PWD:/output docker.io/library/golang:1.18 /bin/bash -c \
+'git clone https://github.com/openshift/hypershift.git && \
+cd hypershift/ && \
+make hypershift && \
+mv bin/hypershift /output/hypershift'
+```
+
```

Comment:
Do we want to ask the user to place the hypershift binary somewhere included in the `$PATH`? , something like: 
`sudo mv $PWD/hypershift /usr/local/bin`

Reply:
fixed